assets/api/schemas/event.js

```diff
@@ -0,0 +1,24 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "id": "http://jsonschema.net",
+  "required": true,
+  "properties": {
+    "eventType": {
+      "type": "number",
+      "required": true
+    },
+    "playlistId": {
+      "type": "number",
```

Comment:
Wouldn't it be better if we used UUIDs?

Reply:
uuid for everything? Well, after reading this I think we can give it a shot: https://blog.codinghorror.com/primary-keys-ids-versus-guids/
(I specially like the part where we don't depend on the DB to generate a new ID)

Reply:
Yep! They're bigger and all, but there's the DB independence + they're really collision-safe and easy to generate/work with